core/ExecutePipe.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "sparta/utils/SpartaAssert.hpp"
 #include "sparta/utils/LogUtils.hpp"
-
+#include "Rename.hpp"
```

Comment:
Why does ExecutePipe need to know about Rename?

Reply:
Wasn't needed, removed in [163a70b](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/163a70b5862658deca0df16b035bbe59dc841c92)